### ISSUE TITLE
0.2 Master your tools

### ISSUE THREAD

**Issue by eda-waffler** [weka-2016/weka-2016#536]
# 0.2 Master your tools
Amongst the other assignments, practice mastering your tools.

- [x] Take a baseline [test](http://www.typingtest.com/index.html) to see how fast you're tying. At a bare minimum you should be able to type 40 WPM by bootcamp.
  - [x] To learn the basics in touch typing and to improve your speeds, try either:
    [Touch Typing Study](http://www.typingstudy.com/lesson/1/part/1) or [TypingClub](http://www.typingclub.com/typing-qwerty-en.html). 
  OR
  - [x] If you are already an accomplished touch typist, use [typing.io](https://typing.io/) and practice typing code, which has a ton of strange symbols.
  - [x] Try [zty.pe](http://zty.pe), but careful: it's addictive! Time box it!
 
- [x] Record your typing speed in the waffle comments below.

**The end goal is to touch type. Even if you've mastered good speed using your own typing style, you need to be able type WITHOUT LOOKING AT THE KEYBOARD!!** Practice touch typing through all coding assignments.  

- [x] Practice your Atom shortcuts.
  - https://www.sitepoint.com/12-favorite-atom-tips-and-shortcuts-to-improve-your-workflow/
  - https://gist.github.com/chrissimpkins/5bf5686bae86b8129bee
  - http://blog.bugsnag.com/atom-editor-cheat-sheet
  - https://www.shortcutworld.com/en/linux/Atom-(text-editor)_1.0.html
  - Got a favourite shortcut resource? Tell us and we'll add it!

- [x] Record the new Atom shortcuts you use in the waffle comments below.
- [x] Record all any new terminal commands you used this week in the waffle comments below.
- [x] Make small commits with concise, readable commit messages.

**Comment by mathieu-anderson** [comment]
Typingtest.com speed 15/10/2016 : 45 wpm (touch typing, which I've started to learn ~1 month ago)

Atom shortcuts are tricky for me, as me computer has a French azerty keyboard, which is not fully supported yet, even with the keyboard-localization package (most glaring flaw is }, which I cannot type in the editor). However, this is ok as Atom auto-closes {, and the issue should be fixed by the 1.12 release in early November, from what I gathered. 

Used (Windows 10):

ctrl-shift-D : duplicate a line
ctrl-up/down : move a line up or down
ctrl-D : select the whole word the selector currently is in. Doesn't select punctuation (. ( { etc)
alt-F3 : select all matching words in the document
ctrl-/ : transform the selected code in a comment
ctrl-l : select a line
tab : select current suggested completion for tag

Commands (Powershell) :
Cd : change directory
Ls : list directories and files in the current directory
Rmdir name : remove directory « name »
Mkdir name : make a new directory called « name »
NewItem -ItemType file name.type : make a new file of such name and type

Git commands (in Powershell as well):
git status : tells me if the elements in the repository are tracked or not, if changes have to be committed, if commits have to be pushed
git add -A : stages every changes made to the repository 
git commit -m "message" : commits the staged changes with a message to describe them
git push : upload the local repository to the GitHub repository
